app/src/lib/stores/app-store.ts

```diff
@@ -1606,9 +1606,22 @@ export class AppStore extends TypedBaseStore<IAppState> {
       pullRequestLabel: this.getPullRequestLabel(repository),
       shellLabel: `Open in ${this.selectedShell}`,
       defaultBranchName: this.getDefaultBranchName(repository),
+      removeRepoLabel: this.getRemoveRepoLabel(),
     })
   }
 
+  private getRemoveRepoLabel() {
+    const showRemoveRepoDialog = this.confirmRepoRemoval
```

Comment:
@Daniel-McCarthy I think this was the missing part from the original PR that avoided those test failures - we didn't _need_ to read local storage because the current value should be store in app state. I'm going to do the same for the `Force Push...` label which is also backed by some app state.

Reply:
Ah, that is a great catch! I'm glad to hear you were able to find a solution to get this to work without breaking the tests!